content/en/integrations/guide/prometheus-metrics.md

```diff
@@ -7,65 +7,134 @@ further_reading:
 - link: "https://www.datadoghq.com/blog/sql-server-metrics/#create-a-stored-procedure-to-generate-and-collect-metrics"
   tag: "Blog"
   text: "Create a stored procedure to generate and collect metrics"
+- link: "/integrations/openmetrics/"
+  tag: "Documentation"
+  text: "Learn about the OpenMetrics integration"
 - link: "/integrations/mysql/"
   tag: "Documentation"
-  text: "Datadog-MySQL integration"
+  text: "Learn about the MySQL integration"
```

Comment:
Can we remove all references to MySQL, SQL Server and SQL stored procedures? Those integrations don't use Prometheus / OpenMetrics nor seem related in any way. Same for the alias (I really wonder how that came to be 🤔 )